### ISSUE TITLE
Redme should describe the intented workflow

### ISSUE THREAD

**Issue by htgoebel** [hawkowl/towncrier#86]
i was suggested to use towncrier as the silver bullet for creating NEWS.rst files. Thus I thought, it will fetch the log messages, magically converts them and creates the NEWS.rst file. Obviously it does not.

As far as I understand, the developer has to create the fragments in the `newsfragments` directory manually. towncrier "only" sorts them (by which criteriea?) and creates the NEWS.rst file.

This workflow needs to be described in the readme.

**Comment by hawkowl** [comment]
Yes, the developer writes the news fragments in the directory (ideally, in more user-focused prose than would exist in the git commit). I'll make a note of this for my docs updates.

**Comment by michaeljoseph** [comment]
@htgoebel tagging #89 as fix for this.
